platform/podjs/README.md

```diff
@@ -20,21 +20,120 @@ feature. While we could support the creation of a container and a
 mechanism for loading features in `pod.js` without too much trouble,
 it is currently not implemented.
 
-The `polyOut` and `polyNav` interfaces are not implemented at the
-moment. The former because we don't use it yet, the latter because it
-is currently not part of the `Pod` interface, which we aim to rectify
-soon.
+## Prerequisites
+
+Make sure you have already installed the following:
+
+-   `node` `v16` or higher
+-   `npm` `v7` or higher
+-   `rollup` `v2.60` or higher
+-   `shx` `v0.3.4` or higher
 
 ## Building
 
-Simply run `build.js` in the repository root.
+Simply run `./build.js` in the repository root.
+
+## How to setup a feature to build with podjs
+
+To create a polyPod compatible feature, you will need to use this module.
+Follow these instructions to do so:
+
+1.  In order to point to `podjs` from the polypoly-eu repository,
+    add as a `development dependency` on your `package.json` the following:
+
+      <code>
+
+         "devDependencies": {
+            "@polypoly-eu/podjs": "file: <path-that-points-to-platform/podjs>",
+         },
+
+      </code>
+
+2.  We use the `rollup` module bundler to build our features all together via the script
+
+    `$ rollup -c`
```

Comment:
Indenting is enough to mark it as code. As above, you can "fence" it marking it as shell to have language-specific highlighting.